x2gbfs/providers/moqo.py

```diff
@@ -39,6 +44,8 @@ class MoqoProvider(BaseProvider):
     DEFAULT_PRICING_PLAN_PATTERN = '{vehicle_type}_hour_daytime'
     MINIMUM_REQUIRED_AVAILABLE_TIMESPAN_IN_SECONDS = 60 * 60 * 3  # 3 hours
 
+    cars_latest_parking_cache: dict[str, int] = {}
```

Comment:
could you add a comment, why this cache is needed?
And is storing the ID as int intended, as for `stations_status` and `station_information` the id is converted to a string.